src/gui_common/ChemicalEquation.cs

```diff
@@ -162,6 +178,19 @@ private void UpdateEquation()
         UpdateEnvironmentPart(environmentalInputs);
     }
 
+    private void UpdateHeader()
+    {
+        spinner.Visible = ShowSpinner;
+        title.Text = EquationFromProcess.Name;
+
+        var colour = DefaultTitleColour;
+
+        if (MarkRedOnLimitingCompounds && EquationFromProcess.LimitingCompounds.Count > 0)
+            colour = new Color(1.0f, 0.3f, 0.3f);
+
+        title.AddColorOverride("font_color", colour);
```

Comment:
```suggestion
        if (MarkRedOnLimitingCompounds && EquationFromProcess.LimitingCompounds.Count > 0){
            title.AddColorOverride("font_color", new Color(1.0f, 0.3f, 0.3f));
        } else {
            title.AddColorOverride("font_color", DefaultTitleColour);
        }
```
(but formatter correctly)